core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -997,19 +995,22 @@ oppia.factory('explorationStatesService', [
       saveFallbacks: function(stateName, newFallbacks) {
         saveStateProperty(stateName, 'fallbacks', newFallbacks);
       },
+      isInitialized: function() {
+        return _states != null;
+      },
       addState: function(newStateName, successCallback) {
         newStateName = $filter('normalizeWhitespace')(newStateName);
         if (!validatorsService.isValidStateName(newStateName, true)) {
           return;
         }
-        if (_states.hasOwnProperty(newStateName)) {
+        if (_states.hasState(newStateName)) {
           alertsService.addWarning('A state with this name already exists.');
           return;
         }
         alertsService.clearWarnings();
 
-        _states[newStateName] = newStateTemplateService.getNewStateTemplate(
```

Comment:
I don't think so, but something to do with newStateTemplateService seems to be causing Travis to fail and I'm still not sure what it is